store/todos.js

```diff
@@ -23,9 +23,8 @@ export const getters = {
 export const mutations = {
   add(state, todo) {
     const newTodo = {
-      id: new Date().getTime(),
+      id: new Date().getTime().toString(),
```

Comment:
Please use a Universally Unique Identifiers to generate ids. You can check out [uuidv4](https://www.npmjs.com/package/uuid4)